lib/code_corps/model/task.ex

```diff
@@ -4,10 +4,12 @@ defmodule CodeCorps.Task do
   import EctoOrdered
 
   alias CodeCorps.Services.MarkdownRendererService
+  alias Ecto.Changeset
 
   @type t :: %__MODULE__{}
 
   schema "tasks" do
+    field :closed_at, :utc_datetime
```

Comment:
I'm not sure I've seen utc_datetime before. Is this new in ecto?

Reply:
The old `:datetime` field now shows deprecation warning on migration.  We need to use either `naive_datetime` or `utc_datetime`.